.github/workflows/ci-weekly.yml

```diff
@@ -160,7 +166,7 @@ jobs:
 
     - name: Install package dependencies
       run: yarn install --frozen-lockfile --ignore-scripts
-
+    
```

Comment:
Ideally, these empty spaces could be reverted as well due to the sensitivity of YAML syntax issues.

Reply:
By design, as empty spaces between steps/tasks make it easier to read.